src/Extra/Pixlr.php

```diff
@@ -185,7 +199,7 @@ public function targetAction(Request $request, $hash, $id)
      */
     public function isEditable(MediaInterface $media)
     {
-        if (!$this->container->get('sonata.media.admin.media')->isGranted('EDIT', $media)) {
```

Comment:
Please keep the container. Otherwise it is a bc break

And only change the url Part